docs/UserGuide.md

```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
 
 ![img.png](DeleteCommandError.png)
 
+### Add remark to a company: `remark` ###
+If you have information not within the fields,
+use the `remark` command to input the additional details.
+
+**Format:** `remark INDEX re/REMARK`
+
+>`INDEX`: Adds remark of the company at INDEX number shown in the displayed company list. </br>
+>`[re/REMARK]` : type prefix `re/` followed by the remark you want to add </br>
+
+
+<div class="typography-box note">
+  <span class="icon">ℹ️</span>
+  <span class="content">
+    <strong>Note :</strong> Remark cannot be empty. Use the unremark command to delete instead.
+  </span>
+</div>
+
+<div class="typography-box warning">
+  <span class="icon">⚠️</span>
+  <span class="content">
+    <strong>Warning :</strong> When adding a remark, the existing remark of the company will be removed i.e adding of remark is not cumulative.
+  </span>
+</div>
+
+**Examples:** </br>
+1. Internship at Google requires knowledge on Java.
+To note down this, type `remark 1 re/need Java`
+![RemarkDemo1.png](images/RemarkDemo1.png)
+
+2. You can view the remark the 1st company as shown below.
+![RemarkDemo2.png](images/RemarkDemo2.png)
+
+**Possible Error:**
+
+If you enter an **empty remark**,
+you will see an error message in the command box. Don’t worry, just add in your remark and
+try the command again!
+An example of an error message is shown below.
+![RemarkErrorMsg.png](images/RemarkErrorMsg.png)
+
+### Delete the remark of a company: `unremark` ###
+If the information under remarks is incorrrect or outdated,
+use the `unremark` command to delete it.
```

Comment:
Similarly, description to be in italics, and more engagement. Use YOU-language LOL